backend/src/LibCloud/SqlCompiler.fs

```diff
@@ -190,11 +190,13 @@ let escapeFieldname (str : string) : string =
 //  evaluation. We expect users to write pure code in here, and if they don't
 //  we can solve that in the next version.
 let rec inline'
+  (fns : Functions)
   (paramName : string)
   (symtable : Map<string, Expr>)
   (expr : Expr)
-  : Expr =
-  RuntimeTypesAst.postTraversal
+  : Ply<Expr> =
+  let identityPly x = uply { return x }
```

Comment:
I'm uncertain whether there's a more effective way to achieve the same result, or if there's a more suitable place for this function

Reply:
I think this is fine